tools/rosbag/src/rosbag/bag.py

```diff
@@ -778,7 +778,7 @@ def __str__(self):
                     else:
                         rows.append(('', s))
         
-        except Exception as ex:
+        except:
```

Comment:
Please do not remove the `Exception` type. This changes behavior and catches other exception not subclasses from `Exception` like `KeyboardInterrupt`.

Same below.